src/bp/ui-shared/src/ConfirmDialog/ConfirmDialog.module.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../node_modules/@blueprintjs/core/src/components/dialog/dialog';
```

Comment:
I wanted it to be as modular as possible so that if we stop having the blueprint css sheet in any of the projects we can still use the components

Reply:
I don't think that's something we will do, it would keep the styling simpler. If we ever want to reuse them somewhere, then we can spend some time on it. 

Otherwise, it's easier to just use the default style

Reply:
fair, will remove it